### PR TITLE
Create pallas.csl

### PR DESCRIPTION
**Pull request by Oxil-2**
CSL style for the French Classics and Ancient History journal "Pallas - Revue d'études antiques", https://journals.openedition.org/pallas/

### CODE REVIEW

File: pallas.csl [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+réalisée par Louis Autin (louis.autin@sorbonne-universite.fr / SU)
+
+ -->
+  <!-- INFORMATIONS RELATIVES A LA FEUILLE DE STYLE -->
+  <info>
+    <title>Pallas (French)</title>
+    <title-short>Pallas</title-short>
+    <id>http://www.zotero.org/styles/pallas</id>
+    <link href="http://www.zotero.org/styles/pallas" rel="self"/>
+    <link href="https://journals.openedition.org/pallas/2624" rel="documentation"/>
```
Comment (adam3smith) [inline pallas.csl:17]: any template style used here?

File: pallas.csl [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+        <names variable="translator"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="auteurs">
+    <names variable="author">
+      <name name-as-sort-order="all" and="text" sort-separator=", " delimiter=", " initialize-with=". " font-variant="small-caps" delimiter-precedes-last="never"/>
+      <label form="short" prefix=" (" suffix=")"/>
+      <substitute>
+        <names variable="editor" font-variant="small-caps"/>
```
Comment (adam3smith) [inline pallas.csl:90]: no need for the font-variant, that's inherited from authors.

File: pallas.csl [hunk 3]
```diff
@@ -0,0 +1,282 @@
...
+        <names variable="translator" font-variant="small-caps"/>
+      </substitute>
```
Comment (adam3smith) [inline pallas.csl:92]: you almost certainly want to add title here to correspond to the in-text citation (also look at the weird webpage example in the preview)

**Review by adam3smith — CHANGES_REQUESTED** [review]
I didn't go line-by-line here, but this has very much the same issues I noted in your stylesheet for Mots, so please also address those. Thanks!

**Comment by Oxil-2** [comment]
Just made all changes accordingly

**Comment by adam3smith** [comment]
Looks great, thanks!

**Pull request merged**